backend/src/main/java/com/benine/backend/Main.java

```diff
@@ -57,27 +48,17 @@ public static void main(String[] args) {
     if(!database.checkDatabase()) //If the database does not exist yet, create a new one
       database.resetDatabase();
     /////
-
+    InetSocketAddress address = new InetSocketAddress(mainConfig.getValue("serverip"),
+            Integer.parseInt(mainConfig.getValue("serverport")));
+    httpController = new HttpController(address, logger, cameraController);
     try {
-      HttpServer server = HttpServer.create(address, 10);
-      server.createContext("/getCameraInfo", new CameraInfoHandler(cameraController));
-      server.createContext("/focus", new FocussingHandler(cameraController));
-      server.createContext("/iris", new IrisHandler(cameraController));
-      server.createContext("/move", new MovingHandler(cameraController));
-      server.createContext("/zoom", new ZoomingHandler(cameraController));
-      server.createContext("/preset", new PresetHandler(cameraController));
-
-      logger.log("Server running at: " + server.getAddress(), LogEvent.Type.INFO);
-      server.start();
       while (true) {
         Thread.sleep(100);
       }
-    } catch (IOException e) {
-      logger.log("Unable to start server", LogEvent.Type.CRITICAL);
     } catch (InterruptedException e) {
-      logger.log("Unable to start server", LogEvent.Type.CRITICAL);
+      e.printStackTrace();
+      logger.log("Exception occured", LogEvent.Type.CRITICAL);
```

Comment:
I don't think this log message is clear enough. `Exception occurred` does not make clear what kind of exception was thrown.